packages/lit-dev-content/site/tutorials/content/async-directive/06.md

```diff
@@ -0,0 +1,54 @@
+The thing we want to do in the timer callback is to update the value rendered by
+the directive -- but notice the timer callback fires _outside_ of the normal
+directive `update`/`render` lifecycle where we normally return a value for
+the directive to render.
+
+Here, we can use `AsyncDirective`'s `setValue` API to update the directive's
+value, outside of the directive's normal update cycle. To do this, we'll need to
```

Comment:
> to do this we need to store

This doesn't seem immediately obvious why. Perhaps move the paragraph that explains this, which is currently under the snippet, right before this section

Reply:
Re-ordered/worded.